### ISSUE TITLE
"Show all automata" doesn't really show all automata...

### ISSUE THREAD

**Issue by martinfabian** [Chalmers-Control-Automation-Mechatronics/Supremica#145]
Attached is a blocking model. Running conflict analysis on it gives a counter-example that can be shown in the Simulator. On the right-click menu in the Simulator there is the "Show all automata" command, which opens up most, but not all automata. The two automata that are not shown are OperatorWins and AddToPot. These two have in common that they consist of a single state with a self-looped transition. I guess this is why they are not shown, but I think this is unwanted optimization. 

The problem created by not showing these two automata is that the blocking looks like a deadlock when it, in fact, is a livelock. In many cases, this distinction is important.

Bottom line: The "Show all automata" should really show all automata.

[Casino-blocking.zip](https://github.com/Chalmers-Control-Automation-Mechatronics/Supremica/files/11342733/Casino-blocking.zip)
